go/main.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/gofiber/fiber"
+	"github.com/gofiber/template/pug"
+	"golang.org/x/oauth2"
+)
+
+/* #nosec G1010*/
+const (
+	accessTokenURL   = "https://api.creditkudos-staging.com/v2/oauth/token"
+	authorizationURL = "https://auth.creditkudos-staging.com/oauth/authorize"
+)
+
+// Update the env in docker with your redirect URL and client id/secret
+var (
+	redirectURL  = ""
+	clientID     = ""
+	clientSecret = ""
+)
+
+func main() {
+	redirectURL = os.Getenv("REDIRECTURL")
+	clientID = os.Getenv("CLIENTID")
+	clientSecret = os.Getenv("CLIENTSECRET")
+	if redirectURL == "" || clientID == "" || clientSecret == "" {
+		panic("Set env variables before running")
+	}
+
+	engine := pug.New("./views", ".pug")
+	app := fiber.New(&fiber.Settings{Views: engine})
+	oauthClient := createOauthClient()
+
+	createRoutes(app, oauthClient)
+
+	if err := app.Listen(8080); err != nil {
```

Comment:
Let's use `3000` for consistency with the other sample and print out where it's listening.

Reply:
When fiber launches it prints out the port. I was unable to get 3000 working in dev as it was `already in use` however 8080 was working.

Reply:
`3000` will work fine if you're not running other stuff.